### PR TITLE
[Improvement]Support multiple receivers.

### PR DESCRIPTION
**Pull request by zqr10159**
## What's changed?

[Improvement]Support multiple receivers.
image
image

## Checklist

- [ ]  I have read the [Contributing Guide](https://hertzbeat.com/docs/others/contributing/)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](../e2e) and all cases have passed.

### CODE REVIEW

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonLongListAttributeConverter.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    @Override
+    public String convertToDatabaseColumn(List<Long> attribute) {
+        return JsonUtil.toJson(attribute);
+
+    }
+
+    @Override
+    public List<Long> convertToEntityAttribute(String dbData) {
+        TypeReference<List<Long>> typeReference = new TypeReference<>() {};
+        return JsonUtil.fromJson(dbData, typeReference);
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonLongListAttributeConverter.java:21]: here we should consider the old db data `long` to convert to `list<Long>` , refer https://github.com/dromara/hertzbeat/blob/master/common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonTagListAttributeConverter.java#L39

File: common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonStringListAttributeConverter.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @Override
+    public String convertToDatabaseColumn(List<String> attribute) {
+        return JsonUtil.toJson(attribute);
+
+    }
+
+    @Override
+    public List<String> convertToEntityAttribute(String dbData) {
+        TypeReference<List<String>> typeReference = new TypeReference<>() {};
+        return JsonUtil.fromJson(dbData, typeReference);
```
Comment (tomsun28) [inline common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonStringListAttributeConverter.java:22]: also here we should consider the old db data `string` to convert to `list<String>`

**Review by tomsun28 — COMMENTED** [review]
LGTM!

**Review by tomsun28 — APPROVED** [review]
LGTM!

**Comment by tomsun28** [comment]
hi, the bot check license header error. 
```
INFO Totally checked 1977 files, valid: 852, invalid: 2, ignored: 1123, fixed: 0 
ERROR the following files don't have a valid license header: 
common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonLongListAttributeConverter.java
common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonStringListAttributeConverter.java 
ERROR one or more files does not have a valid license header 
Error: Process completed with exit code 1.
```

**Comment by zqr10159** [comment]
> hi, the bot check license header error. 
> 
> ```
> 
> INFO Totally checked 1977 files, valid: 852, invalid: 2, ignored: 1123, fixed: 0 
> 
> ERROR the following files don't have a valid license header: 
> 
> common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonLongListAttributeConverter.java
> 
> common/src/main/java/org/apache/hertzbeat/common/entity/manager/JsonStringListAttributeConverter.java 
> 
> ERROR one or more files does not have a valid license header 
> 
> Error: Process completed with exit code 1.
> 
> ```

I will fix it.

**Pull request merged**